.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please don't push your debugging files in merges.

Reply:
Ok, we'll fix them!